scrapers/Vipissy.yml

```diff
@@ -0,0 +1,31 @@
+name: vipissy
+sceneByURL:
+  - action: scrapeXPath
+    url: 
+      - vipissy.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $performer: //dl/dd/a
+    scene:
+      Title: 
+        selector: //section[@class="downloads"]/strong/text()
+        replace:
+          - regex: ^(.*)[—]\s+
+            with:
+      Date:
+        selector: //dl/dd[2]/text()
+        parseDate: Jan 2,2006
+      Details: //div[@class="show_more"]/text()[1]|//div[@class="show_more showed_more"]/text()[1]

```

Comment:
also for the reason  explained in the other scraper `Details: //div[@class="show_more"]/text()[1]` is enough